wrappers/python/openmm/app/gromacstopfile.py

```diff
@@ -1079,8 +1079,11 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
             for pair in pairs:
                 nb.addException(pair[0], pair[1], pair[2], 1.0, 0.0, True)
                 pair_bond.addBond(pair[0], pair[1], [pair[3], pair[4]])
+            existing = set(tuple(lj.getExclusionParticles(i)) for i in range(lj.getNumExclusions()))
```

Comment:
Easier thing to use here is `frozenset`:

```suggestion
             existing = set(frozenset(lj.getExclusionParticles(i)) for i in range(lj.getNumExclusions()))
```

Then you can do `if frozenset(exclusion) not in existing:`